### DISCUSSION TITLE
Monochrome icons for Android 13

### DISCUSSION THREAD

**Discussion by alorma** [community/community#37119] [Mobile]
As documented on Android developer documentation, Android 13 supports tinted icons: https://developer.android.com/develop/ui/views/launch/icon_design_adaptive 

Cold this be added to GitHub app?

**Comment by alorma** [comment]
Sorry, found a similar question...

**Reply by aconsuegra** [reply]
Thanks for bringing it up! We are tracking this request with https://github.com/orgs/community/discussions/32050

**Comment by ossanna16** [comment]
Hi there @alorma and welcome to our community! Thank you for asking a great question 🙂
